src/main/java/com/coursesPlatform/exceptions/ApiExceptionHandler.java

```diff
@@ -0,0 +1,35 @@
+package com.coursesPlatform.exceptions;
+
+import com.coursesPlatform.coursePortfolio.CustomerNotFoundException;
+import com.coursesPlatform.coursePortfolio.MailIsAlreadyInUseException;
+import com.coursesPlatform.coursePortfolio.MailIsInvalidException;
+import com.coursesPlatform.coursePortfolio.TrainerNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.time.LocalDateTime;
+
+@ControllerAdvice
+public class ApiExceptionHandler {
+
+    @ExceptionHandler(value = {CategoryNotFoundException.class, CourseNotFoundException.class, TrainerNotFoundException.class, CustomerNotFoundException.class})
+    ResponseEntity<String> handleNotFoundExceptions(Exception e) {
+        return new ResponseEntity<String>("Not Found!", HttpStatus.NOT_FOUND);
+    }
+
+    @ExceptionHandler(IllegalLengthException.class)
+    ResponseEntity<String> handleIllegalLengthException(Exception e) {
+        return new ResponseEntity<String>("Illegal length !" ,  HttpStatus.BAD_REQUEST);
```

Comment:
I think it should be specified which length is illegal. What is accepted length?